UniMath/CONTENTS.md

```diff
@@ -251,6 +251,11 @@ The packages and files are listed here in logical order: each file depends only
    - [Bicategories/BicategoryFromBicat.v](CategoryTheory/Bicategories/BicategoryFromBicat.v)
    - [Bicategories/BicatFromBicategory.v](CategoryTheory/Bicategories/BicatFromBicategory.v)
    - [Bicategories/Graph.v](CategoryTheory/Bicategories/Graph.v)
+   - [Monoidal/BinaryProductCategories.v](CategoryTheory/Monoidal/BinaryProductCategories.v)
+   - [Monoidal/NaturalIsomorphisms.v](CategoryTheory/Monoidal/NaturalIsomorphisms.v)
```

Comment:
I think this would be fine as a top-level module in `CategoryTheory`, it's my impression that the utility of the concept of a natural isomorphism isn't limited to the study of monoidal categories.